script/.contributors_aliases.json

```diff
@@ -0,0 +1,108 @@
+      "AWhetter@users.noreply.github.com"
+    ]
+  },
+  "Bryce Guinta": {
+    "authoritative_mail": "bryce.paul.guinta@gmail.com",
+    "mails": ["bryce.paul.guinta@gmail.com", "bryce.guinta@protonmail.com"]
+  },
+  "Calen Pennington": {
+    "authoritative_mail": "calen.pennington@gmail.com",
+    "mails": ["cale@edx.org", "calen.pennington@gmail.com"]
+  },
+  "Claudiu Popa": {
+    "authoritative_mail": "pcmanticore@gmail.com",
+    "mails": ["cpopa@cloudbasesolutions.com", "pcmanticore@gmail.com"]
+  },
+  "David Euresti": {
+    "authoritative_mail": "github@euresti.com",
+    "mails": ["david@dropbox.com", "github@euresti.com"]
+  },
+  "Google, Inc.": {
+    "authoritative_mail": null,
+    "mails": [
+      "nathaniel@google.com",
+      "mbp@google.com",
+      "tmarek@google.com",
+      "shlomme@gmail.com",
+      "balparda@google.com",
+      "dlindquist@google.com"
+    ]
+  },
+  "Jacob Bogdanov": {
+    "authoritative_mail": "jacob@bogdanov.dev",
+    "mails": ["jacob@bogdanov.dev", "jbogdanov@128technology.com"]
+  },
+  "Joshua Cannon": {
+    "authoritative_mail": "joshdcannon@gmail.com",
+    "mails": ["joshdcannon@gmail.com", "joshua.cannon@ni.com"]
+  },
+  "Kavins Singh": {
+    "authoritative_mail": "kavinsingh@hotmail.com",
+    "mails": ["kavin.singh@mail.utoronto.ca", "kavinsingh@hotmail.com"]
+  },
+  "Keichi Takahashi": {
+    "authoritative_mail": "keichi.t@me.com",
+    "mails": ["hello@keichi.dev", "keichi.t@me.com"]
+  },
+  "LOGILAB S.A. (Paris, FRANCE)": {
+    "authoritative_mail": "contact@logilab.fr",
+    "mails": [
+      "alexandre.fayolle@logilab.fr",
+      "emile.anclin@logilab.fr",
+      "david.douard@logilab.fr",
+      "laura.medioni@logilab.fr",
+      "anthony.truchet@logilab.fr",
+      "alain.leufroy@logilab.fr",
+      "julien.cristau@logilab.fr",
+      "Adrien.DiMascio@logilab.fr",
+      "emile@crater.logilab.fr",
+      "pierre-yves.david@logilab.fr",
+      "nicolas.chauvat@logilab.fr",
+      "afayolle.ml@free.fr",
+      "aurelien.campeas@logilab.fr",
+      "lmedioni@logilab.fr"
+    ]
+  },
+  "Mario Corchero": {
+    "authoritative_mail": "mariocj89@gmail.com",
+    "mails": ["mcorcherojim@bloomberg.net", "mariocj89@gmail.com"]
+  },
+  "Moises Lopez": {
+    "authoritative_mail": "moylop260@vauxoo.com",
+    "mails": ["moylop260@vauxoo.com"]
+  },
+  "Pierre Sassoulas": {
+    "authoritative_mail": "pierre.sassoulas@gmail.com",
+    "mails": ["pierre.sassoulas@gmail.com", "pierre.sassoulas@cea.fr"]
+  },
+  "Raphael Gaschignard": {
+    "authoritative_mail": "raphael@makeleaps.com",
+    "mails": ["raphael@rtpg.co", "raphael@makeleaps.com"]
+  },
+  "Stefan Scherfke": {
+    "authoritative_mail": "stefan@sofa-rockers.org",
+    "mails": ["stefan.scherfke@energymeteo.de", "stefan@sofa-rockers.org"]
+  },
+  "Sylvain Th\u00e9nault": {
+    "authoritative_mail": "thenault@gmail.com",
+    "mails": ["thenault@gmail.com", "sylvain.thenault@logilab.fr"]
+  },
+  "Ville Skytt\u00e4": {
```

Comment:
Should this be normalised? It is in `CONTRIBUTORS.txt`

Reply:
[dc10b01](https://github.com/PyCQA/astroid/pull/1441/commits/dc10b0184d1565273e4c65389e7fb1b09f4bf003)